suseconnect/migration.go

```diff
@@ -1,5 +1,28 @@
 package main
 
+// TODO LIST
```

Comment:
thanks - breaking down big blocks to smaller functions with structure should help